### ISSUE TITLE
Leadership page is not keyboard accessible

### ISSUE THREAD

**Issue by alexgibson** [mozilla/bedrock#14202]
## Description

It's not possible to navigate through leadership bios using the keyboard.

### Steps to reproduce

1. Open https://www.mozilla.org/en-US/about/leadership/
2. Start tabbing through content

### Expected result

Each leadership bio that expands into a modal with more content should be able to gain keyboard focus. Pressing enter on a focused item should open the modal and move focus to its content.

### Actual result

None of the leadership entries can gain focus, which instead skips to the end of the page content.

### Environment

Firefox Nightly
macOS

**Comment by alexgibson** [comment]
Fixed by https://github.com/mozilla/bedrock/pull/14210
